.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,32 @@
+name: Release drafter
+
+# Push events to every tag not containing "/"
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  draft-a-release:
+    name: Draft a release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifacts
+        uses: dawidd6/action-download-artifact@v2.27.0
+        with:
+          workflow: sql-odbc-main.yml
+          commit: ${{github.sha}}
+          workflow_conclusion: success
```

Comment:
Also will the Release drafter wait until `sql-odbc-main.yml` completes? How does this work?

Reply:
This workflow fails if main CI workflow is not completed yet or failed. But it can be manually restarted to get fresh CI results later.

Reply:
See example in https://github.com/Bit-Quill/sql-odbc/actions/runs/5349394842/jobs/9700551170
Automatic run (attempt 1):
![image](https://github.com/opensearch-project/sql-odbc/assets/88679692/c6ebc4d5-9158-4034-a5b4-11270803a991)
Restarted later (attempt 2):
![image](https://github.com/opensearch-project/sql-odbc/assets/88679692/fe2a62f7-26e8-459e-a979-f4883ab967f7)

Reply:
I see! Thanks! Wondering if something like trigger and wait will be helpful than failing and retrying always?
Because when you push a tag `sql-odbc-main.yml` will run too due to [push](https://github.com/opensearch-project/sql-odbc/blob/main/.github/workflows/sql-odbc-main.yml#L6) event of tag. So I believe it will fail everytime and we need to retry.

Reply:
AFAIK this workflow can uses GHA run triggered by last merge commit on `main`. Cutting a tag on that commit will just re-run it.
I agree that it is not the best option, it needs more testing. I can update it and use [wait-on-check](https://github.com/marketplace/actions/wait-on-check) action later.